tests/test_pps.py

```diff
@@ -78,23 +78,18 @@ def test_datums():
 
     datums = list(sandbox.client.list_datum(job_id))
     assert len(datums) == 1
-    datum = sandbox.client.inspect_datum(job_id, datums[0].datum_info.datum.id)
+    datum = sandbox.client.inspect_datum(job_id, datums[0].datum.id)
     assert datum.state == python_pachyderm.DatumState.SUCCESS.value
 
-    # Skip this check in >=1.11.0, due to a bug:
-    # https://github.com/pachyderm/pachyderm/issues/5123
-    # TODO: remove this check once the bug is fixed
-    if util.test_pachyderm_version() < (1, 11, 0):
-        # Just ensure this doesn't raise an exception
-        sandbox.client.restart_datum(job_id)
+    # TODO: This requires catching a datum while it's being processed
```

Comment:
@Tryneus Hey Marc, I think I'm missing some context, but just from https://github.com/pachyderm/pachyderm/issues/5123, what would normally go in the `DataFilters` field of a cancel request? Why does the datum have to be in progress?

Reply:
It looks like there's several things you can specify as a filter: https://github.com/pachyderm/pachyderm/blob/a2d9ffbff33597be6038decc7c022bdc953dc8b6/src/server/worker/common/common.go#L53

A matching datum does not have to be in-progress, we will just return a `Success: false` if no action was taken.  I don't believe there is any way to test this without a race unless you do something super fancy with a specialized workload.  In this case, I would expect `Success` to always be false because this test waits for the job to finish before attempting to restart it.  `RestartDatum` should never error unless there is a transport issue, from what I can tell.